src/helm/benchmark/run_specs/melt_run_specs.py

```diff
@@ -291,3 +294,109 @@ def get_math_spec(
         + get_generative_harms_metric_specs(),
         groups=["melt"] + groups,
     )
+
+
+@run_spec_function("melt_translation_opus100")
+def get_melt_translation_opus100_spec(language_pair: str, fewshot: bool = False) -> RunSpec:
+    FULL_LANGUAGE_NAMES = {
+        "vi": "Vietnamese",
+        "en": "English",
+    }
+    source_language, target_language = language_pair.split("-")
+
+    scenario_spec = ScenarioSpec(
+        class_name="helm.benchmark.scenarios.melt_translation_scenario.MELTTranslationOPUS100Scenario",
+        args={"source_language": source_language, "target_language": target_language},
+    )
+
+    adapter_spec = get_machine_translation_adapter_spec(
+        source_language=FULL_LANGUAGE_NAMES[source_language],
+        target_language=FULL_LANGUAGE_NAMES[target_language],
+        max_train_instances=5 if fewshot else 0,
+    )
+
+    return RunSpec(
+        name=f"melt_translation_opus100:language_pair={language_pair},fewshot={fewshot}",
+        scenario_spec=scenario_spec,
+        adapter_spec=adapter_spec,
+        metric_specs=get_open_ended_generation_metric_specs(),
+        groups=["melt", "translation_opus100", f"translation_{language_pair}"],
```

Comment:
You may want to have the group name match the run spec name i.e. `melt_translation_opus100`.